### PR TITLE
Complete full group flow, including invite by CSV and individual e-mail...

### PR DESCRIPTION
**Pull request by bportnoy**
...group removal, and admin page.

### CODE REVIEW

File: client/scripts/actions/GroupActions.js [hunk 1]
```diff
@@ -34,9 +41,21 @@ var GroupActions = {
     });
   },
 
+  toggleInviteButtonStatus: function() {
+    Dispatcher.dispatch({
+      actionType: Constants.TOGGLE_GROUP_INVITE_BUTTON_STATUS
```
Comment (ryyann) [inline client/scripts/actions/GroupActions.js]: I've been namespacing constants for easier management. No need to change this now, but FYI.

File: client/scripts/components/modules/admin/CSVPreview.jsx [hunk 2]
```diff
@@ -33,7 +39,7 @@ var CSVPreview = React.createClass({
   },
 
   render: function() {
-    var spinner = this.state.spinner > 0 ? (<i className="fa fa-circle-o-notch fa-spin spinner"></i>) : undefined;
+    var spinner = this.state.spinner ? (<i className="fa fa-circle-o-notch fa-spin spinner"></i>) : undefined;
```
Comment (ryyann) [inline client/scripts/components/modules/admin/CSVPreview.jsx]: FYI, there's a spinner element in widgets/spinner.jsx

File: client/scripts/actions/GroupActions.js (hunk 1 above)
Comment (bportnoy) [inline client/scripts/actions/GroupActions.js]: Yeah, agreed. Was too far down the rabbit hole.

File: client/scripts/components/modules/admin/CSVPreview.jsx (hunk 2 above)
Comment (bportnoy) [inline client/scripts/components/modules/admin/CSVPreview.jsx]: Awesome.

**Comment by ryyann** [comment]
Looks good to me. Merging now.

**Pull request merged**